### PR TITLE
verify_client: refactor this program via an init method

### PR DESCRIPTION
**Pull request by freeHackOfJeff**
1. It's better to isoldate the init_secure_upgrade of sec_api into a method.
2. Use capital letter to define the constant variable
3. Check up boolean type via 'is' rather than '=='

### CODE REVIEW

File: modules/tools/diagnostics/message.py [hunk 1]
```diff
@@ -93,9 +93,9 @@ def callback(self, data):
             time = 0
             sequence_num = 0
 
-        if self.msg_received == True:
+        if self.msg_received:
             seq_diff = sequence_num - self.sequence_num
-            if seq_diff is not 0:
+            if seq_diff == 0:
```
Comment (kernyan) [inline modules/tools/diagnostics/message.py]: Is this change intended? "is not" changed to "=="

**Comment by lianglia-apollo** [comment]
Please add the modified files and re-submit.
